theme_kiddo/views/snippets/s_product_list.xml

```diff
@@ -4,27 +4,29 @@
 <template id="s_product_list" inherit_id="website.s_product_list">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="pt96 pb96" separator=" "/>
+        <attribute name="class" add="pt96 pb96" remove="pt64 pb64" separator=" "/>
     </xpath>
 
     <!-- Container -->
-    <xpath expr="//div[hasclass('container-fluid')]" position="attributes">
-        <attribute name="class" add="container" remove="container-fluid" separator=" "/>
-    </xpath>
     <xpath expr="//div[hasclass('row')]" position="attributes">
         <attribute name="class" add="justify-content-center" separator=" "/>
     </xpath>
+
+    <!-- Title -->
+    <xpath expr="//h2" position="attributes">
+        <attribute name="style">text-align: center;</attribute>
+    </xpath>
     
     <!-- Item n°1 -->
     <xpath expr="//div[hasclass('col-lg-2')]" position="attributes">
         <attribute name="class" add="border rounded mx-3" separator=" "/>
         <attribute name="style">border-radius: 16px !important; border-width: 4px !important; border-style: dotted !important; border-color: var(--o-color-2) !important;</attribute>
     </xpath>
-    <xpath expr="//div[hasclass('s_product_list_item_link')]" position="attributes">
-        <attribute name="class" add="pb24" separator=" "/>
+    <xpath expr="//div[hasclass('col-lg-2')]//h3" position="replace" mode="inner">
+        About us
     </xpath>
-    <xpath expr="//div[hasclass('s_product_list_item_link')]//a" position="replace">
-        <a t-att-href="url">About us</a>
+    <xpath expr="//div[hasclass('col-lg-2')]//div[hasclass('card')]" position="attributes">
+        <attribute name="style">border-width: 0px;</attribute>
```

Comment:
You can therefore remove this, since it will be replaced based on my previous comment :slightly_smiling_face: